lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -33,31 +33,34 @@ module Framework
         # default configurations for the Rails integration; by default
         # the Datadog.tracer is enabled, while the Rails auto instrumentation
         # is kept disabled.
-        DEFAULT_CONFIG = {
-          enabled: true,
-          auto_instrument: false,
-          auto_instrument_redis: false,
-          auto_instrument_grape: false,
-          default_service: 'rails-app',
-          default_controller_service: 'rails-controller',
-          default_cache_service: 'rails-cache',
-          default_grape_service: 'grape',
-          template_base_path: 'views/',
-          tracer: Datadog.tracer,
-          debug: false,
-          trace_agent_hostname: Datadog::Writer::HOSTNAME,
-          trace_agent_port: Datadog::Writer::PORT,
-          env: nil,
-          tags: {}
-        }.freeze
+
+        def self.default_config
```

Comment:
The reason for wrapping the default configuration values inside a method is to defer it's evaluation. This would be addressed by the new configuration API.